### PR TITLE
[CMD] Add Portuguese Brazilian (pt-BR) translation

### PR DESCRIPTION
**Pull request by fernandoisnaldo**
This work was done because I thought there was a translation error, but in fact it was translated into Portuguese from Portugal, based on the Portuguese translation from Portugal and made some changes to create a Brazilian translation.

## Purpose

Create a brazilian Portuguese translate.

JIRA issue: NONE

## Proposed changes

Add brazilian Portuguese translation

### CODE REVIEW

File: base/shell/cmd/lang/pt-BR.rc [hunk 1]
```diff
@@ -0,0 +1,570 @@
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN
```
Comment (binarymaster) [inline base/shell/cmd/lang/pt-BR.rc:1]: New file is added, but you have not included it in common RC file.

Comment (fernandoisnaldo) [reply base/shell/cmd/lang/pt-BR.rc:1]: Well, I'm a layman, but here's a try.

**Comment by JoachimHenze** [comment]
Your first commits headline references the wrong Pull Request and therefore erroneously shows up in a different PR. Can this be fixed by squashing your commits? (I am not sure about that)

**Comment by binarymaster** [comment]
It's not a problem, of course this will be fixed by squashing.

But the actual problem I have commented here earlier is still here.

**Comment by fernandoisnaldo** [comment]
> Your first commits headline references the wrong Pull Request and therefore erroneously shows up in a different PR. Can this be fixed by squashing your commits? (I am not sure about that)

I made several revisions of the same file, that's why it got this mess, I'm sorry for that, because I couldn't find where I delete these edits present in the pull request.

From my contributions, I suggest considering only the commits that appear with the ID
7b1aca5 and 380c334, you can use Ctrl F for this.

File: base/shell/cmd/lang/pt-BR.rc (hunk 1 above)
Comment (HBelusca) [inline base/shell/cmd/lang/pt-BR.rc:1]:
```suggestion
/*
 * PROJECT:     ReactOS CMD Command Line Interpreter
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Portuguese Brazilian resource file
 * TRANSLATORS: Copyright 2023 Fernando Isnaldo Silva de Faria
 */

LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN
```

File: base/shell/cmd/lang/pt-BR.rc [hunk 2]
```diff
@@ -0,0 +1,577 @@
...
+As cores são:\n\
+dec  hex  nome        dec  hex  name\n\
+0    0    Preto       8    8    Cinza (preto brilhante)\n\
+1    1    Azul         9    9    Azul brilhante \n\
+2    2    Verde        10    A    Verde brilhante\n\
+3    3    Ciano         11    B    Ciano brilhante\n\
+4    4    Vermelho     12    C    Vermelho brilhante\n\
+5    5    Violeta      13    D    Violeta brilhante\n\
+6    6    Amarelo      14    E    Amarelo brilhante\n\
+7    7    Branco      15    F    Branco brilhante\n\n"
```
Comment (binarymaster) [inline base/shell/cmd/lang/pt-BR.rc:83]: Columns are wavy...

File: base/shell/cmd/lang/pt-BR.rc [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+                   a condição é falsa\n\
+ERRORLEVEL número  Especifica uma condição verdadeira se a última execução\n\
+                   do programa retornou um código de saída igual ou maior que\n\
+                   o número especificado.\n\
+comando            Especifica o comando a ser executado se a condição for\n\
+                   atendida.\n\
+string1==string2   Especifica uma condição verdadeira se as strings de texto\n\
+                   especificadas coincidirem.\n\
+EXIST arquivo     Especifica uma condição verdadeira se o nome do arquivo\n\
+                   especificado existir.\n\
```
Comment (binarymaster) [inline base/shell/cmd/lang/pt-BR.rc:267]: Indentation between command and description are inconsistent.

**Comment by HBelusca** [comment]
cc @reactos/lang-portuguese

**Review by binarymaster — APPROVED** [review]
Looks like we don't have active Portuguese Brazilian reviewers atm, so I'm going to merge this anyway. It's always possible to make corrections later if needed.

**Pull request merged**